lib/vnet/daemon/client_darwin.h

```diff
@@ -0,0 +1,53 @@
+#ifndef client_darwin_h
+#define client_darwin_h
```

Comment:
FYI, I tend to use Google style for these:

- https://google.github.io/styleguide/cppguide.html#The__define_Guard

TBH I was intending to keep to Google style for most of the native code, but I think I might have strayed at some point.  🤷

Reply:
Thanks, I wasn't aware that there are different styles for these guards. These were automatically generated by Xcode.

Reply:
There are conflicting styles for everything here. Whatever you prefer works.

There's also `#pragma once`, but I don't remember why I didn't do that in the first place in these parts. Probably because following Google Style seemed simpler.